spec/models/articles_db_view_spec.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe ArticlesDbView, type: :model do
+  
+  tag_set = %w(work робота Work Робота)
+  let(:user_data_valid)       do { email: 'admin@example.com' } end
```

Comment:
`let(:user_data_valid) { email: 'admin@example.com' }`

Reply:
Unfortunately without 'do'-'end' wrapper it doesn't work